packages/webview_flutter/tizen/src/webview.cc

```diff
@@ -528,10 +479,20 @@ void WebView::HandleMethodCall(const FlMethodCall& method_call,
     } else {
       result->Error("Invalid argument", "The argument must be a string.");
     }
-  } else if (method_name == "loadRequest") {
-    result->NotImplemented();
-  } else if (method_name == "setCookie") {
-    result->NotImplemented();
+  } else if (method_name == "backgroundColor") {
+    const auto* color = std::get_if<int32_t>(arguments);
+    if (color) {
+      EwkInternalApiBinding::GetInstance().view.SetBackgroundColor(
+          webview_instance_, *color >> 16 & 0xff, *color >> 8 & 0xff,
+          *color & 0xff, *color >> 24 & 0xff);
+      result->Success();
+    }
+  } else if (method_name == "userAgent") {
+    const auto* userAgent = std::get_if<std::string>(arguments);
+    if (userAgent) {
+      ewk_view_user_agent_set(webview_instance_, (*userAgent).c_str());
```

Comment:
nit:
```suggestion
      ewk_view_user_agent_set(webview_instance_, userAgent->c_str());
```

Reply:
I am sorry for late. Thank you for all your review.
I modified it and rebase this PR.